Kinvey/Kinvey/Tests.xcconfig

```diff
@@ -0,0 +1,12 @@
+//
+//  Tests.xcconfig
+//  Kinvey
+//
+//  Created by Victor Hugo Carvalho Barros on 2018-11-14.
+//  Copyright © 2018 Kinvey. All rights reserved.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+SWIFT_SUPPRESS_WARNINGS = YES
```

Comment:
What is this file for?

Reply:
this is a configuration file for the unit tests, since we do test some deprecated methods I added this to remove the warnings for the test bundle